testing/functional_tests/test_automate_training.py

```diff
@@ -10,7 +11,7 @@ def setup_function():
 
 
 @pytest.mark.script_launch_mode('subprocess')
-def test_automate_training(script_runner):
+def test_automate_training(get_functional_test_files, script_runner):
```

Comment:
This might be totally unrelated to this PR but I get this error when running `test_automate_training` locally on PyCharm. The test passes with GitHub Actions, so it seems to be an environment problem of some sort.
```bash
testing/functional_tests/test_automate_training.py:12 (test_automate_training[subprocess])
get_functional_test_files = None, script_runner = <ScriptRunner subprocess>

    @pytest.mark.script_launch_mode('subprocess')
    def test_automate_training(get_functional_test_files, script_runner):
        file_config = os.path.join(__data_testing_dir__, 'automate_training_config.json')
        file_config_hyper = os.path.join(__data_testing_dir__,
                                         'automate_training_hyperparameter_opt.json')
        __output_dir__ = os.path.join(__tmp_dir__, 'results')
    
        ret = script_runner.run('ivadomed_automate_training', '--config', f'{file_config}',
                                '--config-hyper', f'{file_config_hyper}',
                                '--path-data', f'{__data_testing_dir__}',
                                '--output_dir', f'{__output_dir__}')
        print(f"{ret.stdout}")
        print(f"{ret.stderr}")
>       assert ret.success
E       assert False
E        +  where False = <pytest_console_scripts.RunResult object at 0x7fd3831ae400>.success
```

Reply:
I reproduce this EXACT error on my Win10 laptop and desktop so I suspected this was more related to Win10 compatibility before the refactoring and so I was hoping to address this in the Win10 PR down the road. Are you using Win10 as well or different OS? @andreanne-lemay . If you are using anything other than Win10, I will for sure have to look into this much more thoroughly.

Reply:
I am on Linux (Ubuntu 18.04). When I run the test all the files seem to be there, but it still fails :open_mouth: . Also, the other functional tests pass.